Redirecting to top of home page by clicking on footer logo

katarianikita2003 opened the pull request:
On clicking on logo in the footer section, user will be redirected to the top of the home page

(Also press shift+alt+F to format document that's why this much code is updated))

Fixes #727

katarianikita2003 committed:
Redirecting to top of home page by clicking on footer logo

On clicking on logo in the footer section, user will be redirected to the top of the home page

(Also press shift+alt+F to format document that's why this much code is updated))

himanshu-03 commented:
Uploading KrishiConnect _ Shop - Google Chrome 2024-07-02 15-15-38.mp4…

On clicking and trying to hover it glitches like this and this issue is already addressed by someone else. I

katarianikita2003 committed:
Add effect on all pages

himanshu-03 commented:
hey @katarianikita2003 !

Its not visible in any page other than index.html

katarianikita2003 committed:
Merge branch 'main' into main

katarianikita2003 commented:
> hey @katarianikita2003 !
> 
> Its not visible in any page other than index.html

It is working properly in my window

himanshu-03 commented:
Hey @katarianikita2003 !

We have many pages except this too.

himanshu-03 commented:
And if we click on some other pages footer logo it redirects to the top not the homepage as mentioned in your issue and PR

himanshu-03 reviewed (changes requested):
Changes required

katarianikita2003 committed:
Merge branch 'main' into main

katarianikita2003 committed:
Add feature as per request

When click on logo in footer section, user will be redirected to the index.html

RitiChandak commented:
![image](https://github.com/user-attachments/assets/fd114db9-cd86-4fc1-8f72-750fbb96c7e1)
 please resolve

katarianikita2003 committed:
Merge branch 'main' into main

katarianikita2003 commented:
> ![image](https://private-user-images.githubusercontent.com/96969496/356184929-fd114db9-cd86-4fc1-8f72-750fbb96c7e1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMxMTk2MDMsIm5iZiI6MTcyMzExOTMwMywicGF0aCI6Ii85Njk2OTQ5Ni8zNTYxODQ5MjktZmQxMTRkYjktY2Q4Ni00ZmMxLThmNzItNzUwZmJiOTZjN2UxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA4VDEyMTUwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM0ZjkwOThiODc4MDdjZWVjOTllYzZkM2FlYThiZjQyZTEyZGNmYTY2ZDNkYzBmNjU3NmU5ZmViOGZiZDlhMmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Gc_CF7J1Ok3PY1yC1osWW4KcQo26puARJ0B1NccdcGA) please resolve

Done

RitiChandak commented:
![image](https://github.com/user-attachments/assets/a83a783c-f832-4cac-8e8c-309458a2e5be)
resolve this conflicts

RitiChandak commented:
![image](https://github.com/user-attachments/assets/3a3c35aa-1644-41e1-acd3-4f9b990f3276)
Please resolve the conflict
